### PR TITLE
DRILL-8111: Remove lombok usage

### PR DESCRIPTION
**Pull request by vvysotskyi**
# [DRILL-8111](https://issues.apache.org/jira/browse/DRILL-8111): Remove lombok usage

## Description
See ticket for details.

## Documentation
NA

## Testing
NA

### CODE REVIEW

**Comment by luocooong** [comment]
@paul-rogers Could you please take a look? Thank you very much.

File: contrib/storage-http/src/main/java/org/apache/drill/exec/store/http/HttpPaginatorConfig.java [hunk 1]
```diff
@@ -69,6 +60,85 @@
...
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(limitParam, offsetParam, pageParam, pageSizeParam,
+      pageSize, maxRecords, method);
+  }
+
+  @Override
+  public String toString() {
```
Comment (paul-rogers) [inline contrib/storage-http/src/main/java/org/apache/drill/exec/store/http/HttpPaginatorConfig.java:130]: There is a `PlanStringBuilder` that can help with this tedium.

**Review by paul-rogers — COMMENTED** [review]
@luocooong, thanks for doing this boring task. It seems we have quite a few more "data objects" than I expected. Would be nice if there was some code-gen based alternative to Lomboc. Drill already does plenty of code gen and that seems to work fine with IDEs.
 
I didn't carefully check every getter & setter: I assume the tests will catch errors there. The only minor comment is the suggestion to use a specific string builder for any object that might appear in the `EXPLAIN PLAN` output.

File: contrib/storage-http/src/main/java/org/apache/drill/exec/store/http/HttpPaginatorConfig.java (hunk 1 above)
Comment (luocooong) [reply contrib/storage-http/src/main/java/org/apache/drill/exec/store/http/HttpPaginatorConfig.java:130]: @paul-rogers Thanks for the review, this pull request was created by Vova. Anyway, it looks like we'll be able to merge the request soon..

**Comment by vvysotskyi** [comment]
@paul-rogers, thanks for the review, I've addressed the CR comment.

**Review by paul-rogers — APPROVED** [review]
LGTM +1

**Comment by luocooong** [comment]
@vvysotskyi It looks like there are conflicts to resolve before merging, thank you.

**Pull request merged**